src/main/resources/environment.conf

```diff
@@ -7,8 +7,12 @@ pulsar {
     topic=${?PULSAR_CONSUMER_TOPIC}
     subscription="mqtt-gateway"
     subscription=${?PULSAR_CONSUMER_SUBSCRIPTION}
-    subscriptionType="Shared"
+    subscriptionType="Exclusive"
     subscriptionType=${?PULSAR_CONSUMER_SUBSCRIPTION_TYPE}
+    cursor {
+        resetToLatest=false
```

Comment:
the default could be also true in this component. but doesn't matter, can be overridden in envs